TASK_1:
  EDGE: [1, 2]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 80]
  INVOLVED_AGENTS: [1, 2]
  
TASK_2:
  EDGE: [2, 4]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 80]
  INVOLVED_AGENTS: [2, 4]
  
TASK_3:
  EDGE: [1, 3]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [-1.0, 1.5]
  EPSILON: 2.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [40, 80]
  INVOLVED_AGENTS: [1, 3]
  
TASK_4:
  EDGE: [3, 5]
  TYPE: "formation_predicate"
  CENTER: [-1.0, 3.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [80, 80]
  INVOLVED_AGENTS: [3, 5]

TASK_5:
  EDGE: [4, 4]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [1.0, -3.0]
  EPSILON: 1.0
  TEMP_OP: "EventuallyOperator"
  INTERVAL: [50, 80]
  INVOLVED_AGENTS: [4]

TASK_6:
  EDGE: [2, 6]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 8.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [80, 80]
  INVOLVED_AGENTS: [2, 6]

# ----- Phase 2 ----- + 80

TASK_7:
  EDGE: [1, 1]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.0, 0.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [140, 170]
  INVOLVED_AGENTS: [1]
  
TASK_8:
  EDGE: [1, 2]
  TYPE: "formation_predicate"
  CENTER: [0.5, 1.3]
  EPSILON: 1.5
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [160, 170]
  INVOLVED_AGENTS: [1, 2]
  
TASK_9:
  EDGE: [1, 3]
  TYPE: "formation_predicate"
  CENTER: [-1.0, 1.3]
  EPSILON: 1.5
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [150, 170]
  INVOLVED_AGENTS: [1, 3]
  
TASK_10:
  EDGE: [2, 4]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 3.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [140, 170]
  INVOLVED_AGENTS: [2, 4]

TASK_11:
  EDGE: [3, 5]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 4.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [160, 170]
  INVOLVED_AGENTS: [3, 5]
  
TASK_12:
  EDGE: [4, 4]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [1.0, 0.8]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [170, 170]
  INVOLVED_AGENTS: [4]
  
TASK_13:
  EDGE: [5, 5]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.0, 1.5]
  EPSILON: 1.0
  TEMP_OP: "EventuallyOperator"
  INTERVAL: [150, 170]
  INVOLVED_AGENTS: [5]

TASK_14:
  EDGE: [1, 6]
  TYPE: "formation_predicate"
  CENTER: [0.8, 0.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [170, 170]
  INVOLVED_AGENTS: [1, 6]

# ----- Phase 3 -----  + 80 + 90

TASK_15:
  EDGE: [1, 2]
  TYPE: "formation_predicate"
  CENTER: [1.6, 2.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [260, 270]
  INVOLVED_AGENTS: [1, 2]

TASK_16:
  EDGE: [1, 3]
  TYPE: "formation_predicate"
  CENTER: [-1.0, 2.3]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [260, 270]
  INVOLVED_AGENTS: [1, 3]

TASK_17:
  EDGE: [5, 5]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [-0.2, 1.0]
  EPSILON: 0.5
  TEMP_OP: "EventuallyOperator"
  INTERVAL: [210, 270]
  INVOLVED_AGENTS: [5]

TASK_18:
  EDGE: [4, 4]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.3, 1.4]
  EPSILON: 0.5
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [240, 270]
  INVOLVED_AGENTS: [4]

TASK_19:
  EDGE: [1, 6]
  TYPE: "formation_predicate"
  CENTER: [1.5, 1.3]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [260, 270]
  INVOLVED_AGENTS: [1, 6]

TASK_20:
  EDGE: [1, 1]
  TYPE: "go_to_goal_predicate_2d"
  CENTER: [0.0, 0.0]
  EPSILON: 1.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [200, 270]
  INVOLVED_AGENTS: [1]
  
TASK_21:
  EDGE: [2, 4]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.0, 0.0]
  EPSILON: 3.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [230, 270]
  INVOLVED_AGENTS: [2, 4]

TASK_22:
  EDGE: [3, 5]
  TYPE: "epsilon_position_closeness_predicate"
  CENTER: [0.5, 0.7]
  EPSILON: 3.0
  TEMP_OP: "AlwaysOperator"
  INTERVAL: [230, 270]
  INVOLVED_AGENTS: [3, 5]